languages/ruby/exercises/concept/booleans/.docs/introduction.md

```diff
@@ -0,0 +1,60 @@
+## True and False
+
+True and false logical states are represented with `true` and `false` in Ruby. These may either be used as literals on their own, or as a result of logical or comparison methods.
+
+```ruby
+true_variable = true
+false_variable = false
+
+true && false
+# => false
+
+1 < 2
+# => true
+```
+
+## _Truthy_ and _falsey_
+
+When evaluating objects in `if` statements or other boolean contexts, all objects evaluate as _truthy_ **except** for `false` and `nil`.
+
+## Control flow
+
+_Truthy_ and _falsey_ evaluations are useful in the context of control flow. Like in procedural languages, Ruby has an `if`...`else` construct, but it may be more common to use `if` as a "guarding" statement to modify the evaluation of an expression.
+
+```ruby
+def falsey
+  nil || false
+end
+
+def truthy
+  not falsey
+end
```

Comment:
I like this.

Reply:
It was all @kotp for this, I think it makes it really clear